test/dataset/formats/test_coco.py

```diff
@@ -226,6 +232,156 @@ def test_group_coco_annotations_by_image_id(
             ),
             DoesNotRaise(),
         ),  # two image annotations
+        (
+            [
+                mock_cock_coco_annotation(
```

Comment:
Could you please rename `mock_cock_coco_annotation` to `mock_coco_annotation`? :) I just noticed that some refactoring went wrong, and we ended up with this funny name.

Reply:
resolved with a40bf78bb20dcca49ad19c75b808a51d913da9f4